lib/ts/recipe/passwordless/api/implementation.ts

```diff
@@ -97,6 +101,8 @@ export default function getAPIImplementation(): APIInterface {
                     input.options.recipeId +
                     "&preAuthSessionId=" +
                     response.preAuthSessionId +
+                    "&tenantId=" +
+                    input.tenantId +
                     "#" +
                     response.linkCode;
```

Comment:
pass in tenantId to sendEmail in emailpassword and in email verification recipes as well.

Reply:
Done